.github/mergify.yml

```diff
@@ -0,0 +1,43 @@
+pull_request_rules:
+- actions:
+    merge:
+      strict: smart
+      method: merge
+  name: Automatically merge pull requests
+  conditions:
+  - status-success=bench (8.10.2, ubuntu-latest)
+  - status-success=bench (8.6.5, ubuntu-latest)
```

Comment:
Is there a way to avoid hard coding the list of statuses?

Reply:
Unfortunately, I believe it's not possible as they stated in [their document](https://docs.mergify.io/conditions.html#validating-all-status-check)

Reply:
I see, thanks. We'll need to keep an eye on this then